crates/common/tedge_config/src/tedge_config_cli/tedge_config.rs

```diff
@@ -517,10 +517,13 @@ define_tedge_config! {
     mqtt: {
         /// MQTT topic root
         #[tedge_config(default(value = "te"))]
+        #[tedge_config(example = "te")]
         topic_root: String,
 
         /// The device MQTT topic identifier
         #[tedge_config(default(value = "device/main//"))]
+        #[tedge_config(example = "device/main//")]
+        #[tedge_config(example = "device/child-001//")]
```

Comment:
Just a minor tweak to avoid using a `-` as this can result in some unexpected Cumulocity IoT (text-based) searches due to the word based indexing done...not really a thin-edge.io thing, but just to guides c8y users in the right direction.

```suggestion
        #[tedge_config(example = "device/child_001//")]
```

Reply:
Fixed